modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/K6ClientCodegen.java

```diff
@@ -416,26 +512,54 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
 
                 final Operation operation = methodOperation.getValue();
                 final PathItem.HttpMethod method = methodOperation.getKey();
+                OptionalInt operationGroupingOrder = OptionalInt.empty();
+
+                final CodegenOperation cgOperation = super.fromOperation(path, method.name(), operation, null);
+
+                String operationId = operation.getOperationId();
+
+                boolean hasRequestBodyExample = false;
```

Comment:
initialisation of this can go to line 558

Reply:
An outer block needs this variable if I move it inside the `if` block.